packages/suite-data/package.json

```
@@ -22,7 +22,8 @@
         "@trezor/connect": "workspace:*",
         "@trezor/env-utils": "workspace:*",
         "@trezor/urls": "workspace:*",
-        "@trezor/utils": "workspace:*"
+        "@trezor/utils": "workspace:*",
+        "semver": "^7.5.4"
```

this is the current version we use, in other `package.json`.
Doesn't matter much; I see in `yarn.lock` it resolves to the same version.. :shrug:
But to have some order here :policeman:  
```suggestion
        "semver": "^7.7.1",
```